circuits/cpp/barretenberg/cpp/src/barretenberg/honk/sumcheck/sumcheck.hpp

```diff
@@ -162,14 +162,12 @@ template <typename Flavor> class SumcheckVerifier {
     static constexpr size_t MAX_RANDOM_RELATION_LENGTH = Flavor::MAX_RANDOM_RELATION_LENGTH;
     static constexpr size_t NUM_POLYNOMIALS = Flavor::NUM_ALL_ENTITIES;
 
-    VerifierTranscript<FF>& transcript;
     const size_t multivariate_d;
     SumcheckVerifierRound<Flavor> round;
 
     // verifier instantiates sumcheck with circuit size and a verifier transcript
-    explicit SumcheckVerifier(size_t multivariate_n, VerifierTranscript<FF>& transcript)
-        : transcript(transcript)
-        , multivariate_d(numeric::get_msb(multivariate_n))
+    explicit SumcheckVerifier(size_t multivariate_n)
```

Comment:
Updated. I moved the transcript from a class member variable to an input to the method because 1) this is consistent with how it's done in Gemini, Shplonk, etc. and 2) I need this to work for both a native and stdlib transcript which is achieved by either letting the type be auto deduced on input or adding a TemplateType parameter to the class. The latter is ick and also wont be necessary soon once we're only instantiating with std;lib types.